### ISSUE TITLE
Add "Example Playbook" to ansible-galaxy template

### ISSUE THREAD

**Issue by mpdehaan** [ansible/ansible#5397]
I'm noting we're getting some Galaxy playbook examples that are no longer containing examples because it's not in the template (and before people were being overly explanatory in a good way)

I think we should update ansible-galaxy to include an "Examples" section with a dummy playbook in the template, showing passing in some parameters like so:

```
Example Playbook
-------------------------

- hosts: servers
  roles:
     - { role: username.rolename, x: 42 }
```

etc

**Comment by mpdehaan** [comment]
cc @jimi-c

**Comment by jimi-c** [comment]
Resolved by fc3046759345a88436cadfeea5e6570b9eef3e7e.
